### ISSUE TITLE
Suspicious Service Maps Entries

### ISSUE THREAD

**Issue by lucaderi** [ntop/ntopng#5738]
These service maps entries look very suspicious

![Screen Shot 2021-08-06 at 14 05 23](https://user-images.githubusercontent.com/4493366/128507888-da3aa918-ffe7-494c-81eb-bdeed33b0aff.png)

so I click on the host and I get
![Screen Shot 2021-08-06 at 14 05 44](https://user-images.githubusercontent.com/4493366/128508144-d397c2f1-f096-4be8-9086-83c5d5ac55ca.png)

I don't think TCP connections can be established with a 0.0.0.0 host.

**Comment by simonemainardi** [comment]
I managed to reproduce the described behavior. This happens when hosts are serialized by MAC when a MAC generates ARP traffic.

What happens is:
- MAC generates ARP and this creates host 0.0.0.0 inside ntopng
- MAC generates IP traffic and this creates another host != 0.0.0.0, let's say 1.2.3.4, in ntopng
- MAC is seen both as 1.2.3.4 and 0.0.0.0
- When ntopng attempts at labelling MAC with an IP address to nicely draw service maps and tables, it fetches both 1.2.3.4 and 0.0.0.0 and pick one arbitrary IP as label which can be `noip`

So basically things underneath work but the label creates confusion. I have now done an extension to make sure when multiple IP addresses are available, then 0.0.0.0 is not chosen.

**Comment by lucaderi** [comment]
Unable to reproduce it, assuming it is fixed by now.
